index.js

```diff
@@ -1,116 +1,35 @@
-/* eslint max-classes-per-file: ["error", 2] */
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/228172706-5ed0f784-c894-4078-9332-0841cddd2eda.png)

**[OPTIONAL]**

- [ ] It would be great if you remove this Live Demo section because now you have no live link for your project 🤔 so there is no point of having this section 👍 Once you would have a live link then you can add this Live Demo section with a live link in your readme file to make your readme more professional and descriptive 😊